django/test/testcases.py

```diff
@@ -939,6 +939,9 @@ def assertInHTML(self, needle, haystack, count=None, msg_prefix=""):
                 ),
             )
 
+    def assertNotInHTML(self, needle, haystack, msg_prefix=""):
+        self.assertInHTML(needle, haystack, count=0, msg_prefix=msg_prefix)
```

Comment:
What do you think about updating the message when `count` is 0 in `assertInHTML()`?, e.g.
```
f"{needle!r} unexpectedly found in the following response "
f"in the following response\n{haystack_repr}"
```

Reply:
Personally, I like the actual message is because it is very explicit and displays the number of instances found (ie `Found 2 instances (expected 0)`). That being said, I like your suggestion too because it is more concise.

I can modify the message to include your suggestion if you like ?

Reply:
https://github.com/django/django/pull/17626/commits/e193e8b94a9778f089f87ee6a8d4f375403c732c